AD7173.cpp

```diff
@@ -275,3 +275,22 @@ int AD7173Class::set_offset_config(adc7173_register_t offset, uint32_t offset_va
 	/* return error code */
 	return 0;
 }
+
+int AD7173Class::set_gain_registers(adc7173_register_t gain, uint32_t gain_value) {
```

Comment:
Hi, Please add `set_gain_registers` into [keywords.txt](https://github.com/fossasia/neurolab-firmware/blob/master/keywords.txt) file so that it will be recognized as a method by IDE. Thanks

Reply:
Yes. This is a valid request.

Reply:
> Hi, Please add `set_gain_registers` into [keywords.txt](https://github.com/fossasia/neurolab-firmware/blob/master/keywords.txt) file so that it will be recognized as a method by IDE. Thanks

Thanks for the review. corrected it